src/components/Dashboard/TodoList.vue

```diff
@@ -728,11 +730,11 @@ export default class TodoList extends Vue {
 
   /**
    * Checks whether the subject filing is now Completed.
-   * Retries after 1 second for up to 10 iterations.
+   * Retries after 1 second for up to 5 iterations.
    */
   private checkIfCompleted (id: number, count: number): void {
-    // stop this cycle after 10 iterations
-    if (++count > 10) {
+    // stop this cycle after 5 iterations
+    if (++count > 5) {
```

Comment:
Ten iterations should be about 10 seconds... except when the API takes 5 seconds to return from each "GET filing" call, so this mitigates that.